### DISCUSSION TITLE
CSS variables and `Text utility syntax`

### DISCUSSION THREAD

**Discussion by kaitoukid-1412** [tailwindlabs/tailwindcss#12235] [Help] [answered]
![image](https://github.com/tailwindlabs/tailwindcss/assets/15698245/ed940a56-0a3a-446d-a12a-39b3073d2c26)
![image](https://github.com/tailwindlabs/tailwindcss/assets/15698245/5cd91db0-e47f-4482-8b93-ed7f2b1bcdad)

```
https://play.tailwindcss.com/Q5O2wlEMVc
```

```
https://tailwindcss.com/docs/text-color
https://tailwindcss.com/docs/font-size
```

**Comment by adamwathan** [comment, marked as answer]
Hey! Read the docs on arbitrary values for the solution to this:

https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities

**Reply by kaitoukid-1412** [reply]
Thank you @adamwathan
